### ISSUE TITLE
Creating tall elevators causes errors

### ISSUE THREAD

**Issue by FPSUsername** [Pecacheu/Elevators-v2#19]
I created an elevator with two floors. Adding another floor causes this error.

Edit:
This happens when the sign isn't placed on the exact same X and Z coordinates.
The placed sign disappears.

```
[02:11:45] [Server thread/INFO]: [Elevators] DestroyElevSign
[02:12:11] [Server thread/INFO]: [Elevators] Error in fromElevSign: Elevator not detected for sign: CraftBlock{pos=BlockPosition{x=329, y=65, z=-199},type=SPRUCE_WALL_SIGN,data=Block{minecraft:spruce_wall_sign}[facing=north,waterlogged=false],fluid=net.minecraft.server.v1_16_R3.FluidTypeEmpty@6b4283c4}
[02:12:44] [Server thread/INFO]: [Elevators] PreventBDDrop
[02:12:45] [Server thread/INFO]: [Elevators] PreventBDDrop
[02:12:46] [Server thread/INFO]: [Elevators] PreventBDDrop
[02:12:47] [Server thread/INFO]: [Elevators] PreventBDDrop
[02:13:41] [Server thread/INFO]: [Elevators] FROM: 49, TO: 63 (1), STEP: 2.25
[02:14:17] [Server thread/ERROR]: Could not pass event SignChangeEvent to Elevators_v2 v3.2.0
java.lang.ClassCastException: class org.bukkit.craftbukkit.v1_16_R3.block.data.CraftBlockData cannot be cast to class org.bukkit.block.data.type.WallSign (org.bukkit.craftbukkit.v1_16_R3.block.data.CraftBlockData and org.bukkit.block.data.type.WallSign are in unnamed module of loader 'app')
	at com.pecacheu.elevators.Conf.getSignBlock(Conf.java:265) ~[?:?]
	at com.pecacheu.elevators.Conf.addSignBlock(Conf.java:275) ~[?:?]
	at com.pecacheu.elevators.Main.onSignChange(Main.java:143) ~[?:?]
	at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor128.execute(Unknown Source) ~[?:?]
	at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69) ~[patched_1.16.4.jar:git-Paper-399]
	at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80) ~[patched_1.16.4.jar:git-Paper-399]
	at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70) ~[patched_1.16.4.jar:git-Paper-399]
	at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:607) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.PlayerConnection.a(PlayerConnection.java:2813) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.PlayerConnection.lambda$a$18(PlayerConnection.java:2766) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.PlayerConnection.lambda$a$1(PlayerConnection.java:324) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.PlayerConnection.lambda$a$3(PlayerConnection.java:338) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.TickTask.run(SourceFile:18) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeTask(IAsyncTaskHandler.java:136) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandlerReentrant.executeTask(SourceFile:23) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.executeNext(IAsyncTaskHandler.java:109) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.MinecraftServer.bb(MinecraftServer.java:1128) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.MinecraftServer.executeNext(MinecraftServer.java:1121) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.IAsyncTaskHandler.awaitTasks(IAsyncTaskHandler.java:119) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.MinecraftServer.sleepForTick(MinecraftServer.java:1082) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.MinecraftServer.w(MinecraftServer.java:996) ~[patched_1.16.4.jar:git-Paper-399]
	at net.minecraft.server.v1_16_R3.MinecraftServer.lambda$a$0(MinecraftServer.java:173) ~[patched_1.16.4.jar:git-Paper-399]
	at java.lang.Thread.run(Thread.java:834) [?:?]
```

**Comment by Pecacheu** [comment]
80% sure this is one of the issues addressed by the latest build
Will be posting builds soon, it will be working on the latest Paper 1.18.1

**Comment by Pecacheu** [comment]
Issue fixed with v3.3.3
